.github/workflows/ci.yml

```
@@ -0,0 +1,510 @@
+name: Build Action self-tests
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)